[2025-05-12 23:54:28,262] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: api_pipeline_integration.call_api_endpoint scheduled__2025-05-12T23:20:00+00:00 [queued]>
[2025-05-12 23:54:28,268] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: api_pipeline_integration.call_api_endpoint scheduled__2025-05-12T23:20:00+00:00 [queued]>
[2025-05-12 23:54:28,268] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-05-12 23:54:28,268] {taskinstance.py:1357} INFO - Starting attempt 2 of 4
[2025-05-12 23:54:28,268] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-05-12 23:54:28,276] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): call_api_endpoint> on 2025-05-12 23:20:00+00:00
[2025-05-12 23:54:28,279] {standard_task_runner.py:52} INFO - Started process 408 to run task
[2025-05-12 23:54:28,280] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'api_pipeline_integration', 'call_api_endpoint', 'scheduled__2025-05-12T23:20:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/extract_scheduler.py', '--cfg-path', '/tmp/tmp96y9khdt', '--error-file', '/tmp/tmpw_ygq3oj']
[2025-05-12 23:54:28,281] {standard_task_runner.py:80} INFO - Job 7: Subtask call_api_endpoint
[2025-05-12 23:54:28,313] {task_command.py:369} INFO - Running <TaskInstance: api_pipeline_integration.call_api_endpoint scheduled__2025-05-12T23:20:00+00:00 [running]> on host f4c955ecac73
[2025-05-12 23:54:28,360] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_pipeline_integration
AIRFLOW_CTX_TASK_ID=call_api_endpoint
AIRFLOW_CTX_EXECUTION_DATE=2025-05-12T23:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-12T23:20:00+00:00
[2025-05-12 23:54:28,360] {logging_mixin.py:115} INFO - Current working directory: /opt/***
[2025-05-12 23:54:28,360] {python.py:173} INFO - Done. Returned value was: None
[2025-05-12 23:54:28,369] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=api_pipeline_integration, task_id=call_api_endpoint, execution_date=20250512T232000, start_date=20250512T235428, end_date=20250512T235428
[2025-05-12 23:54:28,412] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-12 23:54:28,448] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
